tests/suites/test_suite_rsa.function

```diff
@@ -26,11 +26,17 @@ void rsa_invalid_param()
                                        MBEDTLS_MD_NONE),
                MBEDTLS_ERR_RSA_INVALID_PADDING);
 
+    TEST_EQUAL(mbedtls_rsa_get_padding_mode(&ctx), MBEDTLS_RSA_PKCS_V15);
+    TEST_EQUAL(mbedtls_rsa_get_hash_id(&ctx), MBEDTLS_MD_NONE);
+
```

Comment:
I think checking `MBEDTLS_ERR_RSA_INVALID_PADDING` is sufficient for testing the negative case. So we could move these assertions just after `mbedtls_rsa_init` for testing the getters' functionality.